node/src/accountant/scanners/mod.rs

```diff
@@ -584,6 +580,11 @@ impl PayableScanner {
     fn expose_payables(&self, obfuscated: Obfuscated) -> Vec<QualifiedPayableAccount> {
         obfuscated.expose_vector()
     }
+
+    const ADD_MORE_FUNDS_URGE: &'static str =
+        "Add more funds into your consuming wallet in order to \
+    become able to repay already expired liabilities as the creditors would respond by delinquency \
+    bans otherwise";
```

Comment:
You and I thought it would be reasonable to rename:

`expired liabilities` -> `matured debts`

You'd also like the latter statement to be singular.

`by delinquency bans` -> `by a delinquency ban`